src/object/srv_ec_aggregate.c

```diff
@@ -1468,16 +1463,22 @@ agg_process_holes_ult(void *arg)
 	entry->ae_sgl.sg_nr = 1;
 	entry->ae_sgl.sg_iovs[AGG_IOV_DATA].iov_len = ext_cnt * ext_tot_len *
 								entry->ae_rsize;
```

Comment:
above L1464, why need to "ext_cnt *"? seems "ext_tot_len * entry->ae_rsize" is enough?

Reply:
Fixed.